website/docs/docs/deploy/dbt-cloud-job.md

```diff
@@ -17,17 +17,64 @@ today.
 
 - You must have a [dbt Cloud account](https://www.getdbt.com/signup/) and [Developer seat license](/docs/cloud/manage-access/seats-and-users)
 - You must have a dbt project connected to a [data platform](/docs/cloud/connect-data-platform/about-connections)
-- You must have [access permission](/docs/cloud/manage-access/about-user-access) to create, edit, and run jobs
+- You must [create a dbt Cloud job](#create-and-schedule-jobs)
+- You must have [access permission](/docs/cloud/manage-access/about-user-access) to read, create, edit, or run jobs
 - You must set up a [deployment environment](/docs/collaborate/environments/dbt-cloud-environments) 
 - Your deployment environment must be on dbt version 1.0 or higher
 
 ## Job scheduler features
 
-dbt Cloud's job scheduler, documentation, logging, automation, and alerting features allow you to build observability into your processes easily. 
+dbt Cloud simplifies job scheduling, documentation, logging, automation, and alerting. These features make it easy to deploy jobs confidently and build observability into your processes.
 
-The protection policies on branches ensure that your data moves through governed processes every step of the way, from dev to stage to prod environments.
+### Job scheduler 
 
-### Create and schedule jobs
+The dbt Cloud job scheduler is a powerful tool that helps data teams optimize their data transformation workflows, makes it easier to run dbt in production environments, governs data, automates your dbt jobs, and empowers teams to get results faster with less effort. 
+
+Use the scheduler to improve the efficiency of your data transformation process, while also gaining greater visibility into its progress. You can use custom scheduling for your jobs, which allow you to run them automatically and save money by reducing manual work. It also provides governance policies to make sure your data is transformed consistently and meets your organization's standards.
```

Comment:
😍

Reply:
thank you for that great feedback, Leona!

Reply:
hey @schottj , this is the pr that explains the job scheduler, queue time, and memory pod. let me know your thoughts when you're able to! thank you all